app/elements/demo-tabs.html

```diff
@@ -93,12 +93,12 @@
 
     <div class="layout horizontal">
       <iron-pages selected="[[_selected]]">
-        <content id="contentpanels" select="demo-tab"></content>
+        <slot id="contentpanels" name="demo-tab"></slot>
       </iron-pages>
     </div>
 
     <div class="results">
-      <content select="iframe"></content>
+      <slot select="iframe"></slot>
```

Comment:
this should be `name="iframe"`, no?

Reply:
It _should_ be `<slot name="iframe">` with the corresponding iframes updated with `slot="iframe"`, but since there's no default slot before I'm just going to make this the default (and remove `select="iframe"`).